acceptance/testdata/builder/container/cnb/buildpacks/hello_world/0.0.1/README.md

```diff
@@ -0,0 +1,21 @@
+# Sample: Hello World Buildpack
+
+A no-op buildpack whose intent is to show minimal requirements of a buildpack.
+
+### Usage
+
+```bash
+pack build sample-hello-world-app --builder cnbs/sample-builder:alpine --buildpack .
+```
+
+OR
+
+```bash
+pack build sample-hello-world-app --builder cnbs/sample-builder:bionic --buildpack .
+```
+
+THEN
+
+```
+docker run sample-hello-world-app echo hello
+```
```

Comment:
For maintainability, I think we could delete this file as it's not used anywhere.

Reply:
i deleted two readme file from buildpacks.